app/controllers/katello/api/v2/api_controller.rb

```diff
@@ -126,7 +126,7 @@ def sort_params
 
     def find_optional_organization
       org_id = organization_id
-      return if org_id.nil?
+      return if org_id.blank?
```

Comment:
This also prevents even trying to find an organization with ID `""` (empty string)